### ISSUE TITLE
Administrative metadata only handled for preservation metadata / `digiprovMD`

### ISSUE THREAD

**Issue by prettybits** [keeps/commons-ip#118]
When `METS` files in an E-ARK SIP contain administrative metadata references that aren't preservation metadata, i.e. anything in `rightsMD`, `techMD`, `sourceMD`, validation fails for `CSIP17`, `CSIP31` and `CSIP91` because only the `digiprovMD` section is currently taken into account.

**Comment by carlosjepard** [comment]
Can you please send me a mock SIP so I can test and fix it?

**Comment by prettybits** [comment]
Sorry @carlosjepard , for some reason I missed your message initially.

I mocked up a small example SIP illustrating the issue: [uuid-F353033C-CE32-46F8-9B77-F3A2A372BE27.zip](https://github.com/keeps/commons-ip/files/11102098/uuid-F353033C-CE32-46F8-9B77-F3A2A372BE27.zip)

The mentioned validation rules are all SHOULD, so overall validation still comes out as "valid", but as mentioned, only `digiprovMD` sections are checked against found administrative metadata files in the listed rules.

While experimenting I also found that CSIP17 does _not_ fail as intended when I:
- remove the `dmdSec` while `amdSec` remains present: [uuid-F353033C-CE32-46F8-9B77-F3A2A372BE28.zip](https://github.com/keeps/commons-ip/files/11102355/uuid-F353033C-CE32-46F8-9B77-F3A2A372BE28.zip)
- put non-referenced metadata files in the `metadata/preservation` subfolder: [uuid-F353033C-CE32-46F8-9B77-F3A2A372BE29.zip](https://github.com/keeps/commons-ip/files/11102366/uuid-F353033C-CE32-46F8-9B77-F3A2A372BE29.zip)

I can move these to separate issues if you prefer.
